modules/ROOT/pages/auradb/managing-databases/database-actions.adoc

```diff
@@ -115,42 +120,75 @@ You cannot clone from a Neo4j version 5 instance to a Neo4j version 4 instance.
 === Clone to a new AuraDB instance
 
 . Select the ellipsis (*...*) button on the instance you want to clone.
-. Select *Clone To New* and then *AuraDB* from the resulting menus.
-. Set your desired settings for the new instance. For more information on AuraDB instance creation settings, see xref:auradb/getting-started/create-database.adoc[].
-. Tick the *I understand* checkbox and select *Clone Instance*.
+. Select *Clone To New* and then *AuraDB* from the contextual menu.
+. Set your desired settings for the new database. For more information on AuraDB database creation, see xref:auradb/getting-started/create-database.adoc[].
+. Check the *I understand* box and select *Clone Database*.
++
+[WARNING]
+====
+Make sure that the username and password are stored safely before continuing.
+Credentials cannot be recovered afterwards.
+====
 
-=== Clone to a new AuraDS instance
+=== Clone to an existing AuraDB instance
 
-. Select the ellipsis (*...*) button on the instance you want to clone.
-. Select *Clone To New* and then *AuraDS* from the resulting menus.
-. Set your desired settings for the new instance. For more information on AuraDS instance creation settings, see xref:aurads/create-instance.adoc[].
-. Tick the *I understand* checkbox and select *Clone Instance*.
+When you clone an instance to an existing instance, the URI stays the same, but the data is replaced with the data from the cloned instance.
```

Comment:
Maybe we could specify here `the database connection URI` ?

Reply:
I agree!